src/api/models/assessment.py

```diff
@@ -532,3 +533,37 @@ class AssessmentChange(BaseModel):
 
     def __str__(self):
         return f"{self.event_on} {self.assessment} {self.event_type}"
+
+
+class SurveyQuestion(BaseModel):
+    LOCAL_STAKEHOLDERS = "local stakeholders"
+    GOVERNANCE = "governance"
+    DOMAINS = (
+        (LOCAL_STAKEHOLDERS, _(LOCAL_STAKEHOLDERS)),
+        (GOVERNANCE, _(GOVERNANCE)),
+    )
+
+    key = models.CharField(max_length=255, unique=True)  # migration: populate with current field name
+    domain = models.CharField(max_length=255, choices=DOMAINS)
```

Comment:
Are you maintaining the name 'domains'?

Reply:
Somehow you were looking at a subset of the commits -- the current state of the branch/PR shows everything updated (with `attribute`s among other things).

Reply:
Ok!